cogs/sniper.py

```diff
@@ -123,15 +126,22 @@ async def attempt_snipe(channel, message_cache, message_cache_lock, success_call
                 cache_index += 1
 
         if sniped_user and sniped_content:
+            self.last_snipe = datetime.now()
             await success_callback(channel, sniped_user, sniped_content, sniped_extras, sniped_at)
         else:
-            await Sniper.send_failure_response(channel)
+            await Sniper.send_failure_response(self, message, channel)
 
     @staticmethod
-    async def send_failure_response(channel):
-        embed = create_basic_embed(TEXT_SNIPER_FAIL)
-        embed.set_thumbnail(url=URL_SWIPER_ICON)
-        await channel.send(embed=embed)
+    async def send_failure_response(self, message, channel):
+        time_delta = (datetime.now() - self.last_snipe).total_seconds()
+
+        if time_delta < 5:
+            # TODO: Make emoji configurable
+            await message.add_reaction("<:slowpoke:854520068372168714>")
```

Comment:
define the emoji as a string constant below all the other constants at the top of the file:
`EMOJI_SLOWPOKE = '<:slowpoke:854520068372168714>'`

in general i've been trying to define all user-facing strings/urls/emojis/etc as constants at the tops of their respective files to keep the "user experience" in one place and make it easier to adjust later if needed. not all of the older code is like this, but i'm updating it whenever i touch those files and enforcing this standard for new files

also use single quotes for strings `'` REEEEEEE